registration/registration-services/src/main/java/io/mosip/registration/dao/impl/UserOnboardDAOImpl.java

```diff
@@ -324,7 +352,12 @@ public String insertExtractedTemplates(List<BIR> templates) {
 	}
 
 	private String getBioAttribute(List<String> subType) {
-		String subTypeName = String.join("", subType);
+		String subTypeName;
+		if (subType == null || subType.isEmpty()) {
```

Comment:
Could be replaced with ternary operator.

Reply:
Replaced as suggested.